### PR TITLE
Add a script to run protobrain and fworld together

### PR DESCRIPTION
**Pull request by qemqemqem**

### CODE REVIEW

**Review by wumpusman — COMMENTED** [review]
did you mean to add Fworld? These look like older changes?

**Comment by wumpusman** [comment]
I get the following errors (and this likely relates to if it is meant to be run from obelisk directory): 

! Can't find pats file! This will prevent the model from working! :open pats.json: The system cannot find the file specified.

open brain.cfg: The system cannot find the file specified.
open world.cfg: The system cannot find the file specified.

07-11 12:34:16, Info: Initializing FWorld
07-11 12:34:16, Error: failed to connect

**Comment by qemqemqem** [comment]
Fixed the issue with the pats file.

File: worlds/integrated/fworld/fworld.go [hunk 1]
```diff
@@ -210,7 +209,7 @@ func (ev *FWorld) ConfigPats() {
 		ev.Pats[a] = t
 	}
 
-	ev.OpenPats("pats.json") // TODO add to the .cfg
+	ev.OpenPats("worlds/integrated/fworld/pats.json") // TODO add to the .cfg
```
Comment (jedmccaleb) [inline worlds/integrated/fworld/fworld.go:212]: I don't think we should assume the path like that. everything we need for runs besides the binary should be in the /package dir now

**Comment by garymm** [comment]
Can we close or merge this?

**Pull request closed without merging**